tools/points2labelimage/points2label.py

```diff
@@ -1,232 +1,278 @@
 import argparse
 import json
-import os
 import warnings
 from typing import (
     Dict,
-    List,
+    Optional,
     Tuple,
-    Union,
 )
 
 import giatools.pandas
 import numpy as np
 import numpy.typing as npt
 import pandas as pd
 import scipy.ndimage as ndi
+import skimage.draw
 import skimage.io
 import skimage.segmentation
+from utils import get_list_depth
```

Comment:
Please keep everything in one file. If using Pulsar, Galaxy need to figure out all files that needs to be transferred and every single file needs to be moved, its an unnecessary overhead imho.

Please also add https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-required-files-include